recycler-connect/recyclers
# Global styling updates

## Description
This pull request implements app-wide styling updates, achieving improvements laid out in the project's user stories, specifically issues 11, 14, 15, 16, 17, 18 and 19.

## Related Issues
closes #11 
closes #14
closes #15
closes #16 
closes #17 
closes #18
closes #19

## Updates
### Before
The following screen shot shows the site's styling prior to this PR.
<img width="1552" alt="recyclers before screenshot" src="https://github.com/recycler-connect/recyclers/assets/84364905/a3354420-785e-4473-8b92-764809c9b799">

### After
The following screen shots shows the site's styling with this PR's changes.
<img width="1440" alt="recyclers after screenshot 1" src="https://github.com/recycler-connect/recyclers/assets/84364905/251a9598-0808-44e2-afe1-7bca57fbf2d1">

<img width="1440" alt="recyclers after screenshot 2" src="https://github.com/recycler-connect/recyclers/assets/84364905/09bf5076-edd2-4e83-a5e9-2ff704400db2">

## Acceptance Criteria
Issue #11:
- [x]  WelcomeMessage component is rendered on page load
- [x]  Message describes app to user
- [x]  Message explains how to use the materials filter form to identify recycler matches

Issue #14:
- [x] The message should list how many recyclers matched the user's input. The message may also include information about next steps or where to find further information.

Issue #15:
- [x] ResultsList will map through recyclerResults and render a recyclerCard component for each result
- [x] RecyclerCard will display recycler company name, location and URL link

Issue #16:
- [x] Style app with chosen fonts and colors
- [x]  n/a: Integrate library, if applicable

Issue #17:
- [x] Title and favicon has been updated in index.html
- [x] App title and logo is displayed consistently throughout app

Issue #18:
- [x] MaterialFilter form has been styled
- [x] Form styling matches overall app styling

Issue #19:
- [x] The recyclerCard is styled consistently with the rest of the app's style guidelines

## Testing Steps / QA Criteria
- Visit the deploy preview https://deploy-preview-27--recyclers.netlify.app/
- Ensure that style guidelines are met (colors, fonts, layout).
- Ensure that site content is aligns well with project vision.
